README.md

```diff
@@ -3,6 +3,15 @@
 Welcome to X-Cell Sum app!
 Woo!
 
+# Feature Spec
+* Support positive and negative integers.
+* Ignore blank cells
+* Ignore non-numeric cells. A sum should still be calculated if some cells are text (like “apple”) and other cells are numbers, in the same column.
+* Numbers can be anywhere in the column.
+* Numbers do not have to start at the top of the column.
+* Numbers do not have to be adjacent.
+* You cannot type directly into the sum row. It is not a normal data row, in that sense.
+
 
 # Tech Spec
-[I'm the gdoc of specs] (https://docs.google.com/document/d/18ZCW33AepGB1UdPOQaFnLS_reOJGUPIkQQBimYYovaA/edit)
\ No newline at end of file
+[I'm the gdoc of specs](https://docs.google.com/document/d/18ZCW33AepGB1UdPOQaFnLS_reOJGUPIkQQBimYYovaA/edit)
```

Comment:
That doc does not have its permissions set correctly to allow people who have the link to view it.